frontend/pages/hosts/ManageHostsPage/components/HostsFilterBlock/HostsFilterBlock.tsx

```diff
@@ -315,7 +315,7 @@ const HostsFilterBlock = ({
     }`;
 
     // More narrow tooltip than other MDM tooltip
-    const MDM_STATUS_PILL_TOOLTIP: Record<string, JSX.Element> = {
+    const MDM_STATUS_PILL_TOOLTIP: Record<string, React.ReactNode> = {
```

Comment:
@ghernandez345, it looks like this wasn't actually being used for the filter block (it is using the other tooltip definition instead). Should we remove this or update the filter block to use this narrower version?